tests/NgTests/DnxCatalogCollectorTests.cs

```diff
@@ -88,5 +90,40 @@ public class DnxCatalogCollectorTests
             Assert.Null(otherPackageNuspec.Key);
             Assert.Null(otherPackageNupkg.Key);
         }
+
+        [Theory]
+        [InlineData("1.2.0")]
```

Comment:
please add:
- prerelease versions
- version with 4 integers
- version with leading zeroes

yes we are sort of "re-testing" NuGetVersion but we should cover the basic normalization cases IMO